### PR TITLE
add JavaSE-11

### PR DESCRIPTION
**Pull request by GloriaWa**
add JavaSE-11 to MANIFEST.MF

### CODE REVIEW

File: META-INF/MANIFEST.MF [hunk 1]
```diff
@@ -15,5 +15,6 @@ Require-Bundle: org.eclipse.core.runtime,
  org.eclipse.lsp4j,
  org.eclipse.lsp4j.jsonrpc
 Automatic-Module-Name: LSP_CheckerFramework
-Bundle-RequiredExecutionEnvironment: JavaSE-12
+Bundle-RequiredExecutionEnvironment: JavaSE-12,
```
Comment (wmdietl) [inline META-INF/MANIFEST.MF]: Can you please verify that removing the `JavaSE-12,` here works?

Comment (GloriaWa) [reply META-INF/MANIFEST.MF]: Yes, it works fine for me. I remove the JavaSE-12 in the pr.

**Review by wmdietl — COMMENTED** [review]
Is this the minimum required JDK that users of the plug-in must have?
Would it work to use 8 as the minimum?

**Comment by wmdietl** [comment]
@GloriaWa There are no changes left in this PR. Can you reply to my previous question about using Java 8 as minimum?

**Pull request closed without merging**